jujugui/templates/index.html.mako

```diff
@@ -85,13 +85,21 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
   </head>
 
   <body>
+      <div class="hidden">
+        ## Include the svg sprite image inside the body.
+        <%include file="../static/gui/build/app/assets/stack/svg/sprite.css.svg" />
```

Comment:
This embeds the file into the html, this is so we can reference it with `<use xlink:href=...`